### PR TITLE
Update http api (sub/unsub).

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

File: nanomq/apps/broker.c [hunk 1]
```diff
@@ -754,6 +754,13 @@ proto_work_init(nng_socket sock,nng_socket inproc_sock, nng_socket bridge_sock,
 
 static dbtree *db     = NULL;
 static dbtree *db_ret = NULL;
+static struct hashmap_s *cid_table = NULL;
+
+struct hashmap_s *get_hashmap(void)
```
Comment (alvin1221) [inline nanomq/apps/broker.c:759]: Don't forget to format code.

**Review by alvin1221 — APPROVED** [review]

File: nanomq/apps/broker.c (hunk 1 above)
Comment (lee-emqx) [reply nanomq/apps/broker.c:759]: Thanks for your advance.

**Review by JaylinYu — APPROVED** [review]
wait till we figure out how to deal with subinfol

**Review by wanghaEMQ — COMMENTED** [review]
reviewed

**Pull request merged**